src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotificationsTrait/config.jelly

```diff
@@ -3,4 +3,7 @@
   <f:entry title="${%Communicate Unstable builds to Bitbucket as Successful}" field="sendSuccessNotificationForUnstableBuild">
     <f:checkbox/>
   </f:entry>
+   <f:entry title="${%Do not override PR build statuses}" field="doNotOverridePrBuildStatuses">
+    <f:checkbox/>
+  </f:entry>
```

Comment:
I would like a longer help text that describes what this setting does and why one would want to use it. Yes the information is in <https://github.com/jenkinsci/bitbucket-branch-source-plugin/issues/324>, but it's nicer to have it in Jenkins as well.

Reply:
I meant, in `src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotificationsTrait/help-doNotOverridePrBuildStatuses.html`.

Reply:
thanks, added